src/components/RenderContentObject/KeyFacts.jsx

```diff
@@ -40,7 +40,8 @@ const KeyFacts = (props: tPROPS) => {
       'label': 'FDA SPL files',
```

Comment:
@violetcrestedwren Need "Source of the data" to point to https://dailymed.nlm.nih.gov/dailymed/spl-resources-all-indexing-files.cfm